docs/products/access/api-reference/config-api/identity-provider-api.md

```diff
@@ -151,7 +152,11 @@ Content-Type: application/json
    ],
    "tulip_api_client_id": "accessIntegration",
    "tulip_api_base_url": "https://tulip-api.onewelcome.com/segment",
-   "tulip_api_access_scope": "iwelcome:segment:example"
+   "tulip_api_access_scope": "iwelcome:segment:example",
+   "tulip_api_used_access_methods": [
```

Comment:
Is it `tulip_api_used_auth_methods` or `tulip_api_used_access_methods` (auth vs access)?